ledger/ledger-api-domain/src/main/scala/com/digitalasset/ledger/api/domain.scala

```diff
@@ -255,7 +258,8 @@ object domain {
 
   type EventId = Ref.LedgerString @@ EventIdTag
   val EventId: Tag.TagOf[EventIdTag] = Tag.of[EventIdTag]
-  implicit val eventIdOrdering = scala.math.Ordering.by[EventId, Ref.LedgerString](_.unwrap)
+  implicit val eventIdOrdering: Ordering[EventId] =
+    Ordering.by[EventId, Ref.LedgerString](_.unwrap)
```

Comment:
```scala
    EventId subst implicitly[Ordering[Ref.LedgerString]]
```

Reply:
I just added the type annotation, but I agree that that is very cool.